src/askademic/main.py

```diff
@@ -87,7 +96,10 @@ async def ask_me():
                     )
                 else:
                     pun = allower.data.pun
-                    console.print(f"[bold red]{pun}[/bold red]")
+                    console.print(
```

Comment:
Removed red colour as it's reserved for the errors (e.g. missing key). Plus, added some more info to user othewise they may be confused